spec/services/staff_nomis_checker_spec.rb

```diff
@@ -129,7 +114,7 @@
     end
 
     context 'with slot availability' do
-      context 'with NOMIS_STAFF_SLOT_AVAILABILITY_ENABLED switched OFF' do
+      context 'with NOMIS_STAFF_SLOT_AVAILABILITY_ENABLED switched OFF', vcr: { cassette_name: 'lookup_active_offender', record: :new_episodes } do
```

Comment:
Same here

Reply:
@TobyRet - this context refers to slot availability rather than prisoner availability, so need this one.